src/main/docker/Dockerfile

```diff
@@ -15,21 +15,18 @@
 #
 
 ARG DOCKER_HUB_PUBLIC=docker.io
-FROM ${DOCKER_HUB_PUBLIC}/digitallyseamless/nodejs-bower-grunt:5
+FROM ${DOCKER_HUB_PUBLIC}/cafapi/opensuse-nodejs10:2
 
-# Download and compile v1.29 of tar
-# (Version 1.27.1 is included in the base image but it doesn't have the --exclude-vcs-ignores switch)
-RUN curl -sSL https://ftp.gnu.org/gnu/tar/tar-1.29.tar.gz | tar xzf - -C /usr/src/
-RUN cd /usr/src/tar-1.29/ && FORCE_UNSAFE_CONFIGURE=1 ./configure && make && make install
+RUN zypper install -y gzip && \
+    zypper install -y -t pattern devel_basis
```

Comment:
I'm not sure what the difference is but we normally use `zypper -n install` rather than `zypper install -y`.

Reply:
Updated, '-y' is apparently an alias for '-n':

https://unix.stackexchange.com/a/338314/478638